README.md

```diff
@@ -32,7 +32,7 @@ $ sbt clean package
 [success] Total time: 10 s, completed Feb 21, 2020 12:59:56 PM
 ```
 
-You should have the connector available as `target/scala-2.11/splice-machine-spark-connector_2.11-0.3.0-SNAPSHOT.jar`.
+You should have the connector available as `target/scala-2.11/splice-machine-spark-connector_2.11-0.3.0-SNAPSHOT.jar` for the Spark 2.x versions.  For Spark 3.x, it will be `target/scala-2.12/splice-machine-spark-connector_2.12-0.3.0-SNAPSHOT.jar`.
```

Comment:
nit: Start the sentence `For Spark 3.x` on a new line and markdown will make it look pretty. There are two spaces atm.

Reply:
Changed in commit https://github.com/splicemachine/splice-machine-spark-connector/pull/31/commits/978249160be30c456b08c84acb8e6161476b9410